platform/o.n.bootstrap/src/org/netbeans/JarClassLoader.java

```diff
@@ -77,6 +79,22 @@
  */
 public class JarClassLoader extends ProxyClassLoader {
     private static Stamps cache;
+    private static final String META_INF = "META-INF/";
+    private static final Name MULTI_RELEASE = new Name("Multi-Release");
+    private static final int BASE_VERSION = 8;
+    private static final int RUNTIME_VERSION;
+
+    static {
+        int version;
+        try {
+            Object runtimeVersion = Runtime.class.getMethod("version").invoke(null);
+            version = (int) runtimeVersion.getClass().getMethod("major").invoke(runtimeVersion);
+        } catch (Throwable ex) {
```

Comment:
consider Exception or ReflectiveOperationException please. Catching Throwable without rethrow or handling it can cause all kinds of issues.